piker/brokers/binance.py

```diff
@@ -99,42 +101,67 @@ class Pair(BaseModel):
     quoteOrderQtyMarketAllowed: bool
     isSpotTradingAllowed: bool
     isMarginTradingAllowed: bool
-    
+
     filters: List[Dict[str, Union[str, int, float]]]
     permissions: List[str]
 
 
+# TODO: this isn't being used yet right?
 @dataclass
 class OHLC:
     """Description of the flattened OHLC quote format.
 
     For schema details see:
-        https://binance-docs.github.io/apidocs/spot/en/#kline-candlestick-streams
+    https://binance-docs.github.io/apidocs/spot/en/#kline-candlestick-streams
+
+    documented format:
```

Comment:
If you think we can drop the example (bc self explanatory from API docs) I'm cool with that too.

Reply:
Cool